R/S3_summary.R

```diff
@@ -130,3 +130,18 @@ summarize_tidy <- function(object, test = "t", ...) {
 
   return(tidy_mat)
 }
+
+#' @export
+summary.lh <- function(x,...){
+  print(x)
+  cat("\n")
+  print(x$linearHypothesis)
```

Comment:
summary should not print anything itself. It should return an object to be printed by the REPL. eg `stats::summary.lm` and `stats:::print.summary.lm`

Reply:
sounds good